dlc-manager/tests/channel_execution_tests.rs

```diff
@@ -362,10 +363,13 @@ fn channel_execution_test(test_params: TestParams, path: TestPath) {
             bob_oracles,
             Arc::clone(&mock_time),
             Arc::clone(&electrs),
+            None,
         )
         .unwrap(),
     ));
 
+    let cet_nsequence: u32 = dlc_manager::manager::CET_NSEQUENCE;
```

Comment:
If you just use default value it feels like most of the changes in this file are unnecessary?

Reply:
I'm not sure I really understand what you mean by the 3 values of refund time and nsequence, and I'm not sure how they would be handled in the code.

If you don't mind, would it be OK if I scale this PR back to only be accepting the NUM_CONFIRMATIONS as a configurable variable? That's the only thing I actually wanted, and I feel like I'm starting to bite off more than I can chew.

Reply:
Or feel free to jump in and make commits on this PR if you'd like.